src/constants.rs

```diff
@@ -1,64 +1,64 @@
 pub const WEATHER_CODES: &[(i32, &str)] = &[
-    (113, "☀️"),
-    (116, "🌤️"),
-    (119, "☁️"),
-    (122, "🌥️"),
-    (143, "🌫️"),
-    (176, "🌦️"),
-    (179, "🌧️"),
-    (182, "🌨️"),
-    (185, "🌨️"),
-    (200, "🌩️"),
-    (227, "❄️"),
-    (230, "❄️"),
-    (248, "🌫️"),
-    (260, "🌫️"),
-    (263, "🌧️"),
-    (266, "🌧️"),
-    (281, "🌦️"),
-    (284, "🌦️"),
-    (293, "🌧️"),
-    (296, "🌧️"),
-    (299, "🌧️"),
-    (302, "🌧️"),
-    (305, "🌧️"),
-    (308, "🌧️"),
-    (311, "🌧️"),
-    (314, "🌧️"),
-    (317, "🌧️"),
-    (320, "🌨️"),
-    (323, "🌨️"),
-    (326, "🌨️"),
-    (329, "🌨️"),
-    (332, "🌨️"),
-    (335, "🌨️"),
-    (338, "🌨️"),
-    (350, "🌨️"),
-    (353, "🌧️"),
-    (356, "🌧️"),
-    (359, "🌧️"),
-    (362, "🌨️"),
-    (365, "🌨️"),
-    (368, "🌨️"),
-    (371, "🌨️"),
-    (374, "🌨️"),
-    (377, "🌨️"),
-    (386, "🌩️"),
-    (389, "🌨️"),
-    (392, "🌨️"),
-    (395, "🌨️"),
-    (398, "🌨️"),
-    (401, "🌨️"),
-    (404, "🌨️"),
-    (407, "🌨️"),
-    (410, "🌨️"),
-    (413, "🌨️"),
-    (416, "🌨️"),
-    (419, "🌨️"),
-    (422, "🌨️"),
-    (425, "🌨️"),
-    (428, "🌨️"),
-    (431, "🌨️"),
+    (113, "󰖙"),
+    (116, "󰖕 "),
+    (119, " "),
+    (122, " "),
+    (143, " "),
+    (176, " "),
+    (179, " "),
+    (182, " "),
+    (185, " "),
+    (200, "🌩"),
```

Comment:
no icon for this?

Reply:
there are icons there, wdym?

Reply:
󰖓

Reply:
Doesn't look changed in PR 
![image](https://github.com/user-attachments/assets/c57972eb-f9ce-4599-8e15-786588bba9c6)

Reply:
in either case, I just fixed the non-matching icons, and changes the icons in main.rs too, that way the UI is better unified

Reply:
all the others look good, just these last 2 lightning icons

Reply:
![image](https://github.com/user-attachments/assets/d221d5e2-7f8e-435c-b2ea-0cf102e89897)

Reply:
looks like some extra spaces in there too

Reply:
![image](https://github.com/user-attachments/assets/b745fcaa-b72e-41a5-a063-0bb39cc3c120)

weird

Reply:
How are you grabbing the icons? What font are you using locally and in browser? Is it using the latest nerd fonts ?

Reply:
jetbrains mono nerd, latest, as for browser, floorp's default (noto serf)

as for how I'm grabbing them, kitty's unicode input kitten,